egs2/chime7_task1/asr1/README.md

```diff
@@ -0,0 +1,200 @@
+# CHiME-Dive (CHiME-7 Task 1)
+
+### Distant Automatic Speech Transcription with Multiple Devices in Diverse Scenarios
+
+---
+### Sections
+1. <a href="#description">Short Description </a>
+2. <a href="#data_creation">Data Download and Creation</a>
+3. <a href="#baseline">Baseline System</a>
+4. <a href="#eval_script">Evaluation</a>
+5. <a href="#common_issues">Common Issues</a>
+6. <a href="#reference">References</a>
+
+## <a id="description">1. Short Description  </a>
+
+<img src="https://www.chimechallenge.org/current/task1/images/task_overview.png" width="450" height="230" />
+
+**NOTE: No registration is required at this stage !** 
+
+### <a id="reach_us">Any Question/Problem ? Reach us !</a>
+
+Consider joining the CHiME Google Group and our Slack ! 
+
+
+
+## <a id="installation">2. Installation </a>
+
+**NOTE, if you don't want to use the baseline at all**, you can 
+skip this following procedure and go directly to Section 2.
+For data generation, only the scripts create_dataset.sh, local/data/generate_data.py and 
+local/data/generate_chime6_data.sh are needed, you can git clone ESPNet, and run these without
+ESPNet installation as their dependencies are minimal (`local/data/generate_chime6_data.sh` needs [sox][SoX] however). 
+
+### <a id="espnet_installation">2.1 ESPNet Installation  </a> 
+
+First step, clone ESPNet and checkout the correct git commit hash. <br/>
+(checkout is important to ensure following ESPNet updates will not break the baseline code)
+```bash
+git clone https://github.com/espnet/espnet/
+```
+Next ESPNet must be installed, go to espnet/tools. <br />
+This will install a new environment called espnet with Python 3.9.2
+```bash
+cd espnet
+bash ./tools/setup_anaconda.sh venv espnet 3.9.2
+```
+Activate this new environment.
+```bash
+source ./tools/venv/bin/activate
+conda activate espnet
+```
+Then install ESPNet with Pytorch 1.13.1 be sure to put the correct version for **cudatoolkit**. 
+```bash
+make TH_VERSION=1.13.1 CUDA_VERSION=11.6
+```
+Finally, install other baseline required packages (e.g. lhotse) using this script: 
+```bash
+./local/install_dependencies.sh
+```
+You should be good to go !
+
+**NOTE:** if you encounter any problem have a look at <a href="#common_issues">Section 4. Common Issues</a> here below. <br>
+Or reach us, see <a href="#reach_us">Section 2.</a>
+
+
+## <a id="data_creation">2. Data Download and Creation </a>
+See Data Section in [CHiME-7 Task 1 webpage][] [1] for additional info about the dataset
+and the **rules about the data and pre-trained models**. 
+
+Unfortunately, only 
+
+
+---
+To find out if the data has been generated correctly, you can run this 
+snippet from this directory (assuming you called the challenge dataset dir chime7_task1). <br>
+Your MD5 checksum should check out with ours. 
+```bash
+find chime7_task1 -type f -exec md5sum {} \; | sort -k 2 | md5sum
```

Comment:
Since symbolic links are also included, it shoud be 

```suggest
find chime7_task1 -type f -or -type l -exec md5sum {} \; | sort -k 2 | md5sum
```

I think it's better to distribute a text data containing all values of the md5sum to check which files are invalid instead of the overall sum.

It is also ambiguous whether the evaluation data is included or not.

Reply:
The text file sounds like a good idea.
Yeah it is ambiguous but my original plan was to recompute the hash afterwards also when eval data is released.